src/components/data/EntitiesContent.js

```diff
@@ -377,18 +378,24 @@ const EntitiesContent = ({
       closeOnClick: true,
       content: h(Fragment, [
         h(MenuButton, {
+          onClick: () => setAddingEntity(true)
+        }, 'Add row'),
+        h(MenuButton, {
+          disabled: !entitiesSelected,
+          tooltip: !entitiesSelected && 'Select rows to edit in the table',
           onClick: () => setEditingEntities(true)
-        }, ['Edit Attribute']),
+        }, 'Edit attribute'),
```

Comment:
The general React best practice we follow for children is to contain them in an array. Since there's a single child and it's a string, it'd work fine here without containment but it'd be good to be consistent.

```suggestion
        }, ['Edit attribute']),
```